docs/essentials/app-actions.md

```diff
@@ -90,18 +94,18 @@ catch (FeatureNotSupportedException ex)
 }
 ```
 
-If App Actions are not supported on the specific version of the operating system a `FeatureNotSupportedException` will be thrown. 
+If App Actions aren't supported on the specific version of the operating system a `FeatureNotSupportedException` will be thrown.
 
 The following properties can be set on an `AppAction`:
 
-* Id: A unique identifier used to respond to the action tap.
-* Title: the visible title to display.
-* Subtitle: If supported a sub-title to display under the title.
-* Icon: Must match icons in the corresponding resources directory on each platform.
+- **Id**: A unique identifier used to respond to the action tap.
+- **Title**: the visible title to display.
+- **Subtitle**: If supported a sub-title to display under the title.
+- **Icon**: Must match icons in the corresponding resources directory on each platform.
 
-![App Actions on Homescreen.](images/appactions.png)
+:::image type="content" source="images/appactions.png" alt-text="App actions on home screen":::
```

Comment:
The latest guidance is to add a period at the end of alt-text, which causes screen readers to pause.